sw/airborne/modules/sensors/airspeed_sdp3x.c

```diff
@@ -195,6 +255,9 @@ void sdp3x_event(void)
       // Compute airspeed
       sdp3x.airspeed = sqrtf(Max(sdp3x.pressure * sdp3x.airspeed_scale, 0));
 
+#if USE_AIRSPEED_SDP3X
+      stateSetAirspeed_f(sdp3x.airspeed);
```

Comment:
this should be replaced by something like https://github.com/paparazzi/paparazzi/blob/master/sw/airborne/modules/sensors/airspeed_adc.c#L89

Reply:
Done, and tested on an Apogee v1.0 board